File: src/main/java/group/yunxin/service/BankService.java
package group.yunxin.service;
import java.util.List;

import group.yunxin.pojo.TbBank;
import group.yunxin.vo.PageResult;
/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface BankService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbBank> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum,int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(TbBank bank);
	
	
	/**
	 * 修改
	 */
	public void update(TbBank bank);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbBank findOne(Long id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long [] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(TbBank bank, int pageNum,int pageSize);
	
}

File: src/main/resources/static/js/service/bankService.js
//服务层
app.service('bankService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('/bank/findAll');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('/bank/findPage?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('/bank/findOne?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('/bank/add',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('/bank/update',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('/bank/delete?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('/bank/search?page='+page+"&rows="+rows, searchEntity);
	}    	
});
